### PR TITLE
Add customize options to set flags.

### PR DESCRIPTION
**Pull request by ddoherty03**
Allowing user to set limits on line size, number or results per line,
timeout, and alignment type.  Perhaps the timeout should be set to something
like 5 seconds in case the user enters long execution inadvertently.

### CODE REVIEW

File: seeing-is-believing.el [hunk 1]
```diff
@@ -93,6 +122,13 @@ Optional FLAGS are passed to the seeing_is_believing command."
     (end-of-line)
     (insert " # =>")))
 
+(defun seeing-is-believing~flags ()
```
Comment (jcinnamond) [inline seeing-is-believing.el:125]: Is there a reason why you use `~` in the function name here?

**Review by jcinnamond — COMMENTED** [review]
Thanks for this.

Can you bump the version number at the top of `seeing-is-believing.el` and then I'll merge it.

File: seeing-is-believing.el (hunk 1 above)
Comment (ddoherty03) [reply seeing-is-believing.el:125]: Only to show that it's a "private" function, not meant for the user.  It's a convention I've seen elsewhere, but I'm not dogmatic about it.

**Comment by ddoherty03** [comment]
OK.  Added new commit to bump the version number.  Thanks for looking at this.

**Comment by jcinnamond** [comment]
Great stuff! Thanks.

**Comment by ddoherty03** [comment]
Thank you, John.  I appreciate the merge.

John Cinnamond <notifications@github.com> writes:

> Great stuff! Thanks.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

--
====================================================
Daniel E. Doherty

**Pull request merged**